testing-modules/junit-5/src/test/java/com/baeldung/junit5/testinstance/TweetSerializerJUnit4UnitTest.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.junit5.testinstance;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TweetSerializerJUnit4UnitTest {
```

Comment:
`Minor` - `JUnit4UnitTest` seems to have the word `Unit` in once too many :)

Reply:
As per Baeldung's guidelines, the names of the test classes should end with either 'UnitTest' or 'IntegrationTest'. Hence I haven't changed the name of this class.